core/src/jsTest/kotlin/KeyTests.kt

```diff
@@ -52,4 +52,9 @@ public class KeyTests {
     public fun constructor_withRange_completes() {
         Key(IDBKeyRange.upperBound("foobar", false))
     }
+
+    @Test
+    public fun constructor_withNull_completes() {
+        Key(null)
+    }
```

Comment:
What is this test testing?

Reply:
This PR adds checks in the constructor for `Key` that the values passed in are legal types for IndexedDB. This test is just making sure that the check doesn't throw for `null`.